src/docs/sphinx/dev_guide/testing.rst

```diff
@@ -320,8 +320,8 @@ following items:
 
      and::
 
-       blt_add_test(NAME <example executable name>
-                    COMMAND <example executable name>)
+       axom_add_test(NAME <example executable name>
+                     COMMAND <example executable name>)
```

Comment:
Should we list the other possible options here?
and/or, point to the docs for `blt_add_test` so users know what's available?

Reply:
This was just a bulk rename but I'd rather not just duplicate documentation